app/views/welcome/index.haml

```diff
@@ -0,0 +1,28 @@
+%nav.navbar.navbar-expand-lg.navbar-light.bg-light
+  %a.navbar-brand{:href => "#"} Super Rating Chińczyków
```

Comment:
Consider using current Ruby syntax instead of hash rocket. Also try to follow https://github.com/airbnb/ruby styleguide (empty spaces after/before curly braces).
```
%a.navbar-brand{ href: "#" } Super Rating Chińczyków
```